assets/js/blocks/products/all-products/block.tsx

```diff
@@ -7,10 +7,11 @@ import { InnerBlockLayoutContextProvider } from '@woocommerce/shared-context';
 import { gridBlockPreview } from '@woocommerce/resource-previews';
 import { StoreNoticesContainer } from '@woocommerce/blocks-checkout';
 import { Attributes as ProductListAttributes } from 'assets/js/base/components/product-list/types';
-import { BlockAttributes } from '@wordpress/blocks';
 
 interface BlockProps {
-	attributes: BlockAttributes & ProductListAttributes;
+	attributes: {
+		isPreview?: boolean;
+	} & ProductListAttributes;
```

Comment:
Hi @thealexandrelara, thank you for making the changes. I'm just curious, is it possible to directly add the isPreview type to ProductListAttributes instead of using:
```
	attributes: {
		isPreview?: boolean;
	} & ProductListAttributes;
```

Could we simply have ProductListAttributes include isPreview as one of its attribute types?

Reply:
I chose not to include it in the ProductListAttributes because that interface belongs to a different component that doesn't utilize the isPreview attribute. Since the `isPreview` attribute is only utilized by the All Products block, it made sense to keep it separate from the ProductListAttributes interface.

Reply:
Understood. Thank you for clarifying. I will proceed to approve the pull request. 🚀